comprl-web/app/routes/_dashboard.usr.$name.tsx

```diff
@@ -23,12 +25,21 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
     });
   }
 
-  return {}
+  const user_token = await getToken(user.name);
+
+  if (!user_token) {
+    return { token: "no token exists" };
+  }
+
+  return { token: user_token };
 }
 
 export default function UserDashboard() {
-
+  const { token } = useLoaderData<typeof loader>();
   return (
-    <Typography variant="h1">User</Typography>
+    <>
+      <Typography variant="h1">User</Typography>
+      <Typography variant="body1">Token: {token}</Typography>
```

Comment:
Ye well this is not really the way we can work with it due to web is a bit more complex than adding a headining ... :(
Also the <></> is  pretty sure not the way to go

Reply:
I don't quite understand what you mean by that. How should the token be displayed?

Reply:
I changed it